regaliator.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ['>= 2.1.0', '< 2.5']
 
   spec.add_dependency 'json'
+  spec.add_dependency 'pry'
```

Comment:
Please change this dependency as a `add_development_dependency`.
Pry is not required in production.